### DISCUSSION TITLE
p-calendar does not display the button bar

### DISCUSSION THREAD

**Discussion by ViniciusDevelopment** [primefaces/community#3277] [PrimeNG] [answered]
my **p-calendar** does not display the button bar correctly even though I set it to true the buttons work but they are small and without written names

my code is:
<p-calendar  [showIcon]="true" [touchUI]="true" showButtonBar="true"></p-calendar>

return:

![image](https://github.com/user-attachments/assets/cbd6e103-d7b6-4407-bfdf-8d9bfc468007)

**Comment by ViniciusDevelopment** [comment]
I'm using Angular and Primeng 19

**Reply by qwadrox** [reply, marked as answer]
I guess Calendar is deprecated in Primeng 19, they won't fix the bugs for p-calendar. 
https://primeng.org/guides/migration#compatible
use the identical Date picker instead.

**Reply by qwadrox** [reply]
https://primeng.org/datepicker#buttonbar
